### ISSUE TITLE
[SI-ERROR-FALSE] 3.3.2 Labels or Instructions - Field label is not visible

### ISSUE THREAD

**Issue by cathysnider** [CuBoulder/express#953]
## Problem Description per Siteimprove

The input field does not have a visible label indicating the purpose of the input field. A visible label helps with predictability for all types of users.

## Page Description

Home page of a completely fresh Web Express install. Default content; nothing added so far. User is logged out.

## What we're doing: 

While in Desktop view, this problem occurs with the Mobile Desktop Appliance code.  
< label class="element-invisible" for="edit-search-keys"> Enter the terms you wish to search for. < / label >

## Solution

none suggested. The words 'Search this site' are displayed in the mobile menu, and that should do for sighted users.

**Comment by tovstiadi** [comment]
From a test with a screen reader user on July 1: visually the label is “Search this site” but to a screen reader the message is “Enter the terms you wish to search for”. Why the discrepancy?
